setup.py

```diff
@@ -11,6 +11,7 @@
     "xenon==0.7.0",
     "black==19.10b0",
     "isort==4.3.21",
+    "async_generator==1.10;python_version~='3.6'",
```

Comment:
Should this be `async_generator==1.10;python_version~='3.6.0'`? Currently it's like saying `>=3.6, ==3.*`. https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
Changing to python_version=='3.6.*'